kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/perspective/LibraryPerspective.java

```diff
@@ -66,18 +91,33 @@ public PerspectiveDefinition buildPerspective() {
 
     @OnStartup
     public void onStartup(final PlaceRequest placeRequest) {
+        this.projectPath = (((getWindowParameterMap().containsKey("path")) ? getWindowParameterMap().get("path").get(0) : "")).trim();
```

Comment:
```suggestion
        this.projectPath = (getWindowParameterMap().containsKey("path") ? getWindowParameterMap().get("path").get(0) : "").trim();
```

I think the parentheses are not necessary.